decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -53,12 +53,13 @@ namespace :decidim do
     end
 
     def install_decidim_npm
-      decidim_npm_packages.each do |type, package|
-        if type == :dev
-          system! "npm i -D #{package}"
-        else
-          system! "npm i #{package}"
-        end
+      if decidim_gemspec.source.is_a?(Bundler::Source::Rubygems) && released_version?
+        system! "npm i -D @decidim/dev@~#{decidim_gemspec.version}"
+        system! "npm i @decidim/all@~#{decidim_gemspec.version}"
+      else
+        # Move the Decidim local installer to the Decidim organization
+        system! "npm i https://github.com/mainio/decidim-npm-local"
```

Comment:
I've just did it: https://github.com/decidim/decidim-npm-local :)

Reply:
Changed at bfe571921b455f76405ec8607f7cfaebc6576147.